dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/categories/general/subcategories/production/LksgProductCategory.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.datalandbackend.model.lksg.categories.general.subcategories.production
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import java.math.BigDecimal
+
+/**
+ * --- API model ---
+ * Fields of the LKSG questionnaire regarding a single product category for the "Product Categories" field
+ */
+data class LksgProductCategory(
+    @field:JsonProperty(required = true)
+    val definitionProductTypeService: List<String>,
```

Comment:
name smth like `procurementTypeNaceCode` or similar?

Reply:
I tried to stay as close to the data dictionary as possible with the names. I find `procuredProductTypServiceNaceCodes` a better fit though.

Reply:
Then the data dictionary is bad and should be changed too.
`definitionProductTypeService` is not even close to the actual information that it holds 😆 
Let's discuss and find something better for the model and the data dictionary too.

Reply:
The question reads: "Define the procured product types/services per category (own operations)" and the expected values are Nace Codes. Maybe just "procuredProductTypesAndServices". The name of the field in the Excel is not good, I agree.

Reply:
Changed it to `procuredProductTypesAndServicesNaceCodes` to make usage easier for backend only users. Regarding the backend changes: We should probably reach out to someone on PwC side. For data dictionary relatied issues this is Machthild and Julia?

Reply:
👍  much better in my opinion.

How about we create a sub-task to tell this to Julia (after the daily maybe?) and ask her for "permission" to change this in the data dictionary too?   I am pretty sure she will agree.

Reply:
Aye. Task created.

Reply:
👍 👍